pages/index.js

```diff
@@ -1,7 +1,11 @@
+import Header from "@/components/Header/Header";
+import ToDos from "@/components/ToDos/ToDos";
+
 export default function HomePage() {
   return (
-    <div>
-      <h1>Hello from Next.js</h1>
-    </div>
+    <>
+      <Header />
+      <ToDos />
```

Comment:
As already mentioned in another comment, I suggest to add the `main` element here and to rename your `ToDos` to either `ToDoList` or `TaskList`.